ide/static/js/content.js

```diff
@@ -498,6 +491,80 @@ class Content extends React.Component {
     layer.info.phase = 0;
     this.setState({ net });
   }
+  saveDb(){
+    this.dismissAllErrors();
+    const error = [];
+    const net = this.state.net;
+
+    Object.keys(net).forEach(layerId => {
+      const layer = net[layerId];
+      Object.keys(layer.params).forEach(param => {
+        layer.params[param] = layer.params[param][0];
+        const paramData = data[layer.info.type].params[param];
+        if (layer.info.type == 'Python' && param == 'endPoint'){
+          return;
+        }
+        if (paramData.required === true && layer.params[param] === '') {
+          error.push(`Error: "${paramData.name}" required in "${layer.props.name}" Layer`);
+        }
+      });
+    });
+
+    if (error.length) {
+      this.setState({ error });
+    } else {
+      const netData = JSON.parse(JSON.stringify(this.state.net));
+      Object.keys(netData).forEach(layerId => {
+        delete netData[layerId].state;
+      });
+      this.setState({ load: true });
+      $.ajax({
+        url: '/caffe/save',
+        dataType: 'json',
+        type: 'POST',
+        data: {
+          net: JSON.stringify(netData),
+          net_name: this.state.net_name
+        },
+        success : function (response) {
+          if (response.result == 'success'){
+            prompt('Your prototxt ID is ',response.id);
+          } else if (response.result == 'error') {
+            this.addError(response.error);
+          }
+          this.setState({ load: false });
+        }.bind(this),
+        error() {
+          this.setState({ load: false });
+        }
+      });
+    }
+  }
+  loadDb() {
+    this.dismissAllErrors();
+    const formData = new FormData();
+    const id = prompt('Please enter prototxt id ',id);
+    formData.append('proto_id', id);
+    $.ajax({
+      url: '/caffe/load',
+      dataType: 'json',
+      type: 'POST',
```

Comment:
This can easily be changed to a GET right? To support the pastebin functionality we discussed?